jdat_notebooks/environment.yml

```diff
@@ -23,9 +23,10 @@ dependencies:
   - pyregion
   - webbpsf>=0.8
   - scikit-learn
-  - lmfit
+  - lmfit>=0.9
   - specutils
   - dill
+  - dust-extinction
```

Comment:
Technically, all requirements should exist in your `cube_fitting/requirements` file -- so no need to add it here in the global environment file.

```suggestion
```

Reply:
Done